handlers/external/external_test.go

```diff
@@ -472,22 +472,22 @@ func TestSending(t *testing.T) {
 
 	var jsonChannel30IntLength = courier.NewMockChannel("8eb23e93-5ecb-45ba-b726-3b064e0c56ab", "EX", "2020", "US",
 		map[string]interface{}{
-			"send_path":                     "",
-			"max_length":                    30,
-			courier.ConfigSendBody:          `{ "to":{{to}}, "text":{{text}}, "from":{{from}}, "quick_replies":{{quick_replies}} }`,
-			courier.ConfigContentType:       contentJSON,
-			courier.ConfigSendMethod:        http.MethodPost,
-			courier.ConfigSendAuthorization: "Token ABCDEF",
+			"send_path":               "",
+			"max_length":              30,
+			courier.ConfigSendBody:    `{ "to":{{to}}, "text":{{text}}, "from":{{from}}, "quick_replies":{{quick_replies}} }`,
+			courier.ConfigContentType: contentJSON,
+			courier.ConfigSendMethod:  http.MethodPost,
+			courier.ConfigSendHeaders: map[string]interface{}{"Authorization": "Token ABCDEF", "foo": "bar"},
 		})
 
 	var xmlChannel30IntLength = courier.NewMockChannel("8eb23e93-5ecb-45ba-b726-3b064e0c56ab", "EX", "2020", "US",
 		map[string]interface{}{
-			"send_path":                     "",
-			"max_length":                    30,
-			courier.ConfigSendBody:          `<msg><to>{{to}}</to><text>{{text}}</text><from>{{from}}</from><quick_replies>{{quick_replies}}</quick_replies></msg>`,
-			courier.ConfigContentType:       contentXML,
-			courier.ConfigSendMethod:        http.MethodPost,
-			courier.ConfigSendAuthorization: "Token ABCDEF",
```

Comment:
should make sure one of these tests is still using the old `send_authorization` key

Reply:
@jcbalmeida can you just make sure one of these is using the old config key and then we'll get this merged

Reply:
Yes, we made a data migration in RapidPro to keep it compatible to existing channels: https://github.com/Ilhasoft/rapidpro/blob/f75d9a4d81c7af41d6ea7929f3b0674e4666145f/temba/channels/migrations/0136_external_headers.py#L6-L18

The PR for RapidPro is https://github.com/nyaruka/rapidpro/pull/3589

Reply:
I mean tweak one of these tests so we know it still works with the old config, until that migration is run

Reply:
I pushed a commit just now with this tweak to ensure compatibility